modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -600,6 +600,15 @@ CV__DNN_INLINE_NS_BEGIN
         static Ptr<RegionLayer> create(const LayerParams& params);
     };
 
+    /**
+     * @brief \f$ L_p \f$ - detection output layer.
+     *
+     * num() and channels() are 1.
```

Comment:
What is num and channels? Just specify how output shape is computed

Reply:
OK

Reply:
Is it better like this:

/**
     * @brief detection output layer.
     *
     * The layer size is: @f$ (1 \times 1 \times N \times 7) @f$
     *    where N is the number of detections after nms, and each row is:
     *    [image_id, label, confidence, xmin, ymin, xmax, ymax]
     */

Reply:
number of detections multiplied by batch size?

Reply:
Yes. At least that's what I see in code.